### PR TITLE
In create contest

### PR DESCRIPTION
**Pull request by UVStudio**
Completed Create-Contest feature

### CODE REVIEW

File: client/src/components/createContestComponents/CreateContest.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+export const CreateContest = () => {
+  const classes = useStyles();
+  const contest = useContext(ContestContext);
+
+  const title = contest.title;
+  const description = contest.description;
+  const prize = contest.prize;
+  const deadline = contest.selectedDate;
+  const contestPics = contest.pics;
```
Comment (bonnieli) [inline client/src/components/createContestComponents/CreateContest.js]:
```suggestion
  const {title, description, prize, selectedDate: deadline, pics: contestPics} = contest;
```

can take advantage of es6 deconstruction methods to really shorten the num lines of code here :)

File: client/src/imgDataS3/index.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo15.png',
+    title: 'tattoo15',
+    cols: 1,
+  },
+  {
+    img:
+      'https://team-owl-tattoo.s3.ca-central-1.amazonaws.com/tattoos/tattoo16.png',
```
Comment (bonnieli) [inline client/src/imgDataS3/index.js:94]: i see you stored these in s3, you can delete the local ones

File: client/src/components/createContestComponents/CreateContest.js (hunk 1 above)
Comment (UVStudio) [reply client/src/components/createContestComponents/CreateContest.js]: This has been updated. Good call. Thanks.

File: client/src/imgDataS3/index.js (hunk 2 above)
Comment (UVStudio) [reply client/src/imgDataS3/index.js:94]: Done. There is still the empty folder. I'll get rid of that once this is merged. Thanks!

**Pull request merged**